apps/frontend/src/app/components/header/header.component.scss

```diff
@@ -207,6 +207,9 @@
     }
     a {
       display: inline-block;
+      @media screen and (max-width: 992px) {
+        padding-right: 176.79px;
+      }
```

Comment:
I think a cleaner solution would be to set the width to 100% and the box-sizing to border-box, because if we decide some day that we should change the width of the drawer then we don't need to change this. Also if the kréta logo changes then we only need to change the file and nothing else.